xml/deployment_hwrecommend.xml

```diff
@@ -154,7 +154,7 @@
       <para>
        Set the cluster network using the following command:
       </para>
-<screen>&prompt.root;ceph config set osd cluster_network <replaceable>MY_NETWORK</replaceable></screen>
+<screen>&prompt.root;ceph config set global public_network <replaceable>MY_NETWORK</replaceable></screen>
```

Comment:
Actually, this should be `global cluster_network`, no? @smithfarm

Reply:
Yes